Joinpoint Regression in R.R

```diff
@@ -61,133 +73,158 @@ ggplot(mortdata, aes(year, asdr, color = cod)) +
-
+# arguments: number of deaths, population size
+# (denominator of mortality rate), time
+# the output of this fit is super straightforward:
+# we have a y-intercept of 46.75... and a slope of -0.027...
+# you can index these values easily as well
+fit.1 <- ljr0(mdw$pi.n, mdw$pop, mdw$year)
+
+# arguments: number of deaths, population size, time
+# summ parameter automatically print summary table
+# now, we have a y-int of -15.61... and some other things!
+# g0 is the slope of the line before any joinpoints are identified.
+# g1 max(t-tau1,0)
+# is the slope of the line after the joinpoint is identified.
+# we also have our first tau (joinpoint) estimate: 1918!
+fit.2 <- ljr1(mdw$pi.n, mdw$pop, mdw$year, summ = TRUE)
+
+# arguments: number of joinpoints to find (k),
+# number of deaths, population size, time
+# y-intercept: -27.9...
+# g0: slope from 1900-1918: 0.0115
+# g1 max(t-tau1,0), slope from 1919-1956: 0.0337
+# g2 max(t-tau2,0), slope from 1957-1998: -0.058
+# what this means is that after 1956 P&I deaths decreased a lot!
 fit.3 <- ljrk(2, mdw$pi.n, mdw$pop, mdw$year)
-        # arguments: number of joinpoints to find (k), number of deaths, population size, time
-        # y-intercept: -27.9...
-        # g0: slope from 1900-1918: 0.0115
-        # g1 max(t-tau1,0), slope from 1919-1956: 0.0337
-        # g2 max(t-tau2,0), slope from 1957-1998: -0.058
-        # what this means is that after 1956 P&I deaths decreased a lot!
 
+# arguments: number of joinpoints to find (k),
+# number of deaths, population size, time
+# If you ran this model, you likely noticed how much SLOWER it was.
+# You can ask the model to fit as many joinpoints as your heart desires.
+# But you have to think about whether this is a smart thing to do or
+# not, and what it would mean to fit all those points. It would end up
+# being likely pretty meaningless. So take a look at this output and see
+# if you can interpret it.
 fit.4 <- ljrk(3, mdw$pi.n, mdw$pop, mdw$year)
-        # If you ran this model, you likely noticed how much SLOWER it was. 
-        # You can ask the model to fit as many joinpoints as your heart desires. But you have to think about
-        # whether this is a smart thing to do or not, and what it would mean to fit all those points. It would end up
-        # being likely pretty meaningless. So take a look at this output and see if you can interpret it. 
 
 
-# Since we want to be able to do all of this more quickly without writing four different models for every cause of death,
-    # let's write a function instead that will do everything we want, including extracting those tau estimates
-    # so we can easily index them for data visualization
+
+# Since we want to be able to do all of this more quickly without writing four
+# different models for every cause of death,
+# let's write a function instead that will do everything we want,
+# including extracting those tau estimates
+# so we can easily index them for data visualization
```

Comment:
I think leaving function definitions here is fin, as opposed to begging of the script, however the arguments should be described and the returns made explicit in the comments.